Source/ProgressionSystemRuntime/Private/Data/PSWorldSubsystem.cpp

```diff
@@ -9,6 +9,7 @@
 #include "Components/StaticMeshComponent.h"
 #include "Data/PSDataAsset.h"
 #include "Data/PSSaveGameData.h"
+#include "Engine/CurveTable.h"
```

Comment:
is not used

Reply:
removed: https://github.com/moinm3uw/ProgressionSystem/pull/57/commits/57f6ad94305a0fcde2e5895f3801c295b470846f